python/nano/test/run-nano-test-ipex.sh

```diff
@@ -15,6 +15,7 @@ now=$(date "+%s")
 time1=$((now-start))
 
 echo "#2 Start test trainer ipex"
+pip install neural-compressor
 start=$(date "+%s")
 python -m pytest -s ${NANO_TEST_DIR}/test_trainer_ipex.py
 now=$(date "+%s")
```

Comment:
@yangw1234 Temporary installation to test INC. I will open another issue to refactor the test.

Reply:
Will move installation to jenkins configure.